Starting new node with 0 cache raises error

At the time of starting a new node with `--cache 0` an error raises when trying to create Bloom filter:

```
panic: failed to create bloom: m (number of bits in the Bloom filter) must be >= 2

goroutine 1 [running]:
github.com/ethereum/go-ethereum/trie.NewSyncBloom(0x0, 0x7fcb6439f8f0, 0xc0005ae4c0, 0x7fcb6439f8f0)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/trie/sync_bloom.go:73 +0x28b
github.com/ethereum/go-ethereum/eth.NewProtocolManager(0xc00050e460, 0x0, 0x1, 0x137608855a, 0xc0003c37d0, 0x184e340, 0xc000324000, 0x7fcb6439ee28, 0xc0003d87e0, 0xc000936000, ...)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/eth/handler.go:169 +0x7a5
github.com/ethereum/go-ethereum/eth.New(0xc0005ae660, 0xc000376600, 0x423f78, 0x30, 0x1536d00)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/eth/backend.go:253 +0xd5f
github.com/ethereum/go-ethereum/cmd/utils.RegisterEthService.func2(0xc0005ae660, 0xc0000ad260, 0xc00039d940, 0xc0005ae620, 0x2)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/cmd/utils/flags.go:1757 +0x4f
github.com/ethereum/go-ethereum/node.(*Node).Start(0xc00063e000, 0x0, 0x0)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/node/node.go:216 +0x463
github.com/ethereum/go-ethereum/cmd/utils.StartNode(0xc00063e000)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/cmd/utils/cmd.go:67 +0x2f
main.startNode(0xc000400c60, 0xc00063e000)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/cmd/geth/main.go:366 +0x10f
main.geth(0xc000400c60, 0x0, 0x0)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/cmd/geth/main.go:341 +0x127
github.com/ethereum/go-ethereum/vendor/gopkg.in/urfave/cli%2ev1.HandleAction(0x136c360, 0x16aed88, 0xc000400c60, 0xc000382f60, 0x0)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/vendor/gopkg.in/urfave/cli.v1/app.go:490 +0x82
github.com/ethereum/go-ethereum/vendor/gopkg.in/urfave/cli%2ev1.(*App).Run(0xc0002b2000, 0xc000136000, 0x2d, 0x30, 0x0, 0x0)
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/vendor/gopkg.in/urfave/cli.v1/app.go:264 +0x5f5
main.main()
	/home/runner/work/quorum/quorum/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/cmd/geth/main.go:280 +0x55
```

--- a/docker-compose/alastria-node-data/env/geth.common.sh
+++ b/docker-compose/alastria-node-data/env/geth.common.sh
@@ -9,7 +9,7 @@ ISTANBUL_REQUESTTIMEOUT="10000"
 # Blockchain sync mode
 SYNCMODE="fast"
 # Cache size in MB
-CACHE="0"
+CACHE="8"
 # Blockchain garbage collection mode
 GCMODE="full"
 # General logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail

